Altis_Life.Altis/core/fn_setupEVH.sqf

```diff
@@ -12,3 +12,4 @@ player addEventHandler["Take",{_this call life_fnc_onTakeItem}]; //Prevent peopl
 player addEventHandler["Fired",{_this call life_fnc_onFired}];
 player addEventHandler["InventoryClosed", {_this call life_fnc_inventoryClosed}];
 player addEventHandler["InventoryOpened", {_this call life_fnc_inventoryOpened}];
+player addEventHandler["HandleRating",{0}];
```

Comment:
Look at lie 12, 11, and 9 you fucking moron and then spew shit about "consistency"